### DISCUSSION TITLE
VirtualInput ios 9 有没有兼容，兼容写法应该怎么写

### DISCUSSION THREAD

**Discussion by 15239939087** [ant-design/ant-design-mobile#5905] [General]
VirtualInput ios 9 有没有兼容，兼容写法应该怎么写

**Comment by miracles1919** [comment]
<img width="747" alt="image" src="https://user-images.githubusercontent.com/22469543/211448868-4f101cf5-5b4f-4479-ad48-e9cc0026d804.png">
